app/src/main/res/values-v21/styles.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
```

Comment:
Why is this change required?

Reply:
otherwise app crashes, pls read pr desc :-)

Reply:
Oops missed that. 😅

Reply:
np 😀